codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/AWSHttpBindingProtocolGenerator.kt

```diff
@@ -44,14 +44,20 @@ abstract class AWSHttpBindingProtocolGenerator : HttpBindingProtocolGenerator()
     override val shouldRenderCodingKeysForEncodable = true
 
     override fun generateProtocolUnitTests(ctx: ProtocolGenerator.GenerationContext): Int {
+        val ignoredTests = setOf(
+            // Glacier customizations
```

Comment:
I think we can remove `// Glacier customizations` -- it's obvious based on the name of the test :)

Reply:
removed